### PR TITLE
IOW-135 - statistical gw daily data

### PR DESCRIPTION
**Pull request by dsteinich**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/main/resources/mybatis/monitoringLocation.xml [hunk 1]
```diff
@@ -3,25 +3,24 @@
...
-               site_name,
-               to_char(st_y(geom),'FM90.0000000') latitude,
-               to_char(st_x(geom),'FM990.0000000') longitude
-          from (select monitoring_location_identifier,
-                       site_name,
-                       st_transform(geom, 4326) geom
-                  from nwis.monitoring_location
-                 where geom is not null and
-                       monitoring_location_identifier = #{value,jdbcType=VARCHAR}) site_data
+    <select id="getGeoJson" parameterType="String" resultType="String">
```
[the remark this answers is not in the document]
Comment (dsteinich) [reply src/main/resources/mybatis/monitoringLocation.xml]: This is not part of a UI

File: src/main/java/gov/usgs/wma/waterdata/timeseries/geojson/TimeSeriesProperties.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import java.time.LocalDate;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import gov.usgs.wma.waterdata.geojson.Properties;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@JsonPropertyOrder(value={"observationType","phenomenonTimeStart","phenomenonTimeEnd",
+		"observedPropertyName","observedPropertyReference","samplingFeatureName",
+		"samplingFeatureName","statistic","statisticReference",
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/wma/waterdata/timeseries/geojson/TimeSeriesProperties.java]: Might be an extra `samplingFeatureName` right here.

File: src/main/java/gov/usgs/wma/waterdata/timeseries/geojson/TimeSeriesProperties.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+		"timeStep","unitOfMeasureName","unitOfMeasureReference",
+		"result","nilReason","approvals",
+		"qualifiers","grades"})
+public class TimeSeriesProperties extends Properties {
+
+	@Schema(description="The type of this observation - Currently only 'MeasureTimeseriesObservation' objects are produced.", example="MeasureTimeseriesObservation")
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/wma/waterdata/timeseries/geojson/TimeSeriesProperties.java]: These example attributes are pretty cool.

File: src/main/resources/mybatis/monitoringLocation.xml [hunk 4]
```diff
@@ -3,25 +3,24 @@
...
+        select json_build_object(
```
Comment (ssoper-usgs) [inline src/main/resources/mybatis/monitoringLocation.xml]: This is a cool postgres feature!

**Pull request merged**